frame/mosaic-vault/src/mocks/runtime.rs

```diff
@@ -0,0 +1,258 @@
+
+use crate::{self as mosaic_vault, *};
+//use crate as mosaic_vault;
+use super::currency_factory::MockCurrencyId;
+use super::*;
+// use crate::*;
+use frame_support::{
+    ord_parameter_types,
+    construct_runtime,parameter_types,
+    traits::{Everything, GenesisBuild, UnixTime},
+    PalletId,
+};
+use sp_keystore::{testing::KeyStore, SyncCryptoStore};
+use frame_system::EnsureSignedBy;
+use frame_system as system;
+use orml_traits::parameter_type_with_key;
+use num_traits::Zero;
+use sp_core::{sr25519::Signature, H256};
+use sp_runtime::{
+	testing::{Header, TestXt},
+	traits::{BlakeTwo256, ConvertInto, Extrinsic as ExtrinsicT, IdentifyAccount, IdentityLookup, Verify, Saturating},
+	RuntimeAppPublic,
+};
+
+use std::time::{SystemTime, UNIX_EPOCH};
+
+
+pub type BlockNumber = u64;
+pub type Balance = u128;
+pub type Amount = i128;
+pub type Nonce = u128;
+pub type TransferDelay = u128;
+pub type VaultId = u64;
+pub type DepositId = [u8; 32];
+pub type RemoteNetworkId = u64;
+pub type AccountId = u128;
+
+pub const MINIMUM_BALANCE: Balance = 1000;
+// accounts 
+pub const RELAYER_ACCOUNT: AccountId = 0;
+pub const ALICE: AccountId = 1;
+pub const BOB: AccountId = 2;
+pub const CHARLIE: AccountId = 4;
+pub const JEREMY: AccountId = 5;
+pub const FEE_ADDRESS: AccountId = 777;
+pub const ACCOUNT_FREE_START: AccountId = JEREMY + 1;
+pub const ACCOUNT_INITIAL_AMOUNT: u128 = 1_000_000;
+
+pub const MILLISECS_PER_BLOCK: u64 = 6000;
+
+parameter_types!{
+    pub const BlockHashCount: u64 = 250;
+}
+
+impl frame_system::Config for Test {
+    type AccountId = AccountId;////u64;
+    type Origin = Origin;
+    type Index = u64;
+    type BlockNumber = BlockNumber;
+    type Call = Call;
+    type Hash = H256;
+    type Hashing = sp_runtime::traits::BlakeTwo256;
+    type Lookup = IdentityLookup<Self::AccountId>;
+    type Header = Header;
+    type Event = Event;
+    type BlockHashCount = BlockHashCount;
+    type BlockWeights = ();
+    type BlockLength = ();
+    type Version = ();
+    type PalletInfo = PalletInfo;
+    type AccountData = pallet_balances::AccountData<Balance>;
+    type OnNewAccount = ();
+    type OnKilledAccount = ();
+    type DbWeight = ();
+    type BaseCallFilter = Everything;
+    type SystemWeightInfo = ();
+    type SS58Prefix = ();
+    type OnSetCode = ();
+}
+
+parameter_type_with_key! {
+	pub ExistentialDeposits: |_currency_id: MockCurrencyId| -> Balance {
+		Zero::zero()
+	};
+}
+
+impl orml_tokens::Config for Test {
+    type Event = Event;
+    type Balance = Balance;
+    type Amount = Amount;
+    type CurrencyId = MockCurrencyId;
+    type WeightInfo = ();
+	type ExistentialDeposits = ExistentialDeposits;
+	type OnDust = ();
+	type MaxLocks = ();
+	type DustRemovalWhitelist = Everything;
+}
+
+parameter_types! {
+	pub const ExistentialDeposit: u64 = 1000;
+}
+
+// impl pallet_balances::Config for Test {
```

Comment:
please remove comment or comment why it is commented

Reply:
removed